examples/tools/cmake/cmake_toolchain/local_flow.rst

```diff
@@ -0,0 +1,83 @@
+CMakeToolchain in the developer flow
```

Comment:
I personally don't see this so much as an example but as a part of the tutorial in the creating packages section where we explain with details what is the developer flow vs doing a `conan create` and